acceptance/build_info_test.go

```diff
@@ -30,7 +30,11 @@ import (
 func TestBuildInfo(t *testing.T) {
 	l := localcluster.Create(1, stopper)
 	l.Start()
-	defer l.Stop()
+	defer func() {
+		if err := l.AssertAndStop(nil); err != nil {
```

Comment:
This would be cleaner as `defer l.AssertAndStop(t)`. You could avoiding making `localcluster` depend on `testing` by providing a small interface:

```
type logger interface {
  Fatal(args ...interface{})
}
```